Info-section

MartinL-no opened the pull request:
The following changes have been made to meet the acceptance criteria -

- Added how it works and info section
- Added links to 'profy.dev'  and 'click for more information' text

MartinL-no committed:
Add info sections

MartinL-no committed:
Add 'profy.dev' and 'Click here for more information.' links

MartinL-no committed:
install react-router-hash-link

MartinL-no committed:
Add smooth scroll to 'how it works' and 'about' sections

MartinL-no committed:
Merge branch 'main' into info-section

mentor-tara commented:
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime, consider starting with the next task already to practice the professional Git workflow. [If you need a refresher, feel free to follow this section in the course again.](https://profy.dev/project/reddit-timer/while-waiting)[](ckmadiu6g3nlc0a84dmaspn6b)

MartinL-no committed:
bug fix

mentor-tara reviewed (changes requested):
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] Use the `article` tag instead of `section` [:link:](https://profy.dev/project/reddit-timer/review-tips/prefer-article-over-section)

**Styles:**
- [x] **Refrain from styling children with CSS selectors** [:link:](https://profy.dev/project/reddit-timer/review-tips/css-selectors)
- [x] **Extract shared CSS values** [:link:](https://profy.dev/project/reddit-timer/review-tips/common-css-values-in-theme)
- [x] Use global styles when appropriate (e.g. for font-size of headings, color of links...) [:link:](https://profy.dev/project/reddit-timer/review-tips/globally-style-elements)

**Tests:**
- [x] **Use [within](https://testing-library.com/docs/dom-testing-library/api-helpers#within-and-getqueriesforelement-apis) to test multiple occurences of elements (e.g. the profy.dev link)** [:link:](https://profy.dev/project/reddit-timer/review-tips/testing-library-within)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/info-section-54645480-9f74-4a7a-b1c0-a98de4445a2a). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckmbnzw4843z70b41h7aujv0o)

MartinL-no committed:
Make requested changes

- Add any duplicated styling to global styles
- Change section tags to article
- Add tests (including within functionality)

MartinL-no committed:
Add @testing-library/dom to dependencies and fix formatting issues in App.test.js

MartinL-no committed:
formatting fix

mentor-tara reviewed (approved):
Great work! :tada::tada::tada: You can merge the PR now.[](ckmadn6tc3o1v0b87a2kyd9yb)

### Note: This is the last task included in the free membership.

It's up to you of course, but I'd be super happy to finish this project with you.

**_This is what you'll get when you upgrade to a premium membership:_**

- Get access to the tasks and designs for the search page.
- Build a dynamic form to fetch data from an API.
- Transform the data as preparation for the UI.
- Build the interactive heatmap.
- Write automated tests for dynamic components.
- Get advice on how to prepare this project for your job hunt.

I hope you're ready to take the leap. This is where it gets real.

# [Go Premium Now](https://profy.dev/#premium)

See you soon 😄 

> Note: If you're already a premium member, you can continue right away, of course.[](ckmadn6tc3o1v0b87a2kyd9yb)
